apps/web/app/layout.tsx

```diff
@@ -1,22 +1,21 @@
-import "./globals.css";
-import type { Metadata } from "next";
-import { Inter } from "next/font/google";
+import type React from 'react';
+import { Container, CssBaseline } from '@mui/material';
+import MainBar from '@repo/ui/mainbar';
 
-const inter = Inter({ subsets: ["latin"] });
-
-export const metadata: Metadata = {
-  title: "Create Turborepo",
-  description: "Generated by create turbo",
-};
+import './globals.css';
 
 export default function RootLayout({
-  children,
+	children,
 }: {
-  children: React.ReactNode;
-}): JSX.Element {
-  return (
-    <html lang="en">
-      <body className={inter.className}>{children}</body>
-    </html>
-  );
+	children: React.ReactNode;
+}) {
```

Comment:
promote/make Props a type variable, here is a template that should be use everywhere for function component:
```typescript
import type { FC, ReactNode } from 'react';

// then we could easily export interface Props if we need to
interface Props {
  children: ReactNode,
}

export const RootLayout: FC<Props> = ({ children }) => {
  ...
};

export default RootLayout;
```